sysbuild/sdp.cmake

```diff
@@ -2,18 +2,29 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
-# If it is enabled, include the SDP GPIO application in the build
-if(SB_CONFIG_SDP_GPIO)
+if(SB_CONFIG_SDP)
   # Extract SoC name from related variables
   string(REPLACE "/" ";" split_board_qualifiers "${BOARD_QUALIFIERS}")
   list(GET split_board_qualifiers 1 target_soc)
   set(board_target_flpr "${BOARD}/${target_soc}/cpuflpr")
   set(target_soc)
 
+  # Select the SDP application
+  if(SB_CONFIG_SDP_GPIO)
+    set(sdp_app_dir "${ZEPHYR_NRF_MODULE_DIR}/applications/sdp/gpio")
```

Comment:
this really needs to come from a Kconfig like the network core image so people can add their own, will approve but fix in a follow up PR